src/platforms/dialogflow/DialogflowEvent.ts

```diff
@@ -95,26 +92,20 @@ export class DialogflowEvent extends VoxaEvent {
    */
   protected initUser(): void {
     const { originalDetectIntentRequest } = this.rawEvent;
-    const { conv } = this.google;
+    const { conv } = this.dialogflow;
     const storage = conv.user.storage as any;
     let userId: string = "";
 
-    this.source = _.get(originalDetectIntentRequest, "source") || "google";
-
-    if (this.source === "google") {
-      if (conv.user.id) {
-        userId = conv.user.id;
-      } else if (_.get(storage, "voxa.userId")) {
-        userId = storage.voxa.userId;
-      } else {
-        userId = v1();
-      }
-
-      _.set(this.google.conv.user.storage, "voxa.userId", userId);
-    } else if (this.source === "facebook") {
-      userId = _.get(originalDetectIntentRequest, "payload.data.sender.id");
+    if (conv.user.id) {
+      userId = conv.user.id;
+    } else if (_.get(storage, "voxa.userId")) {
+      userId = storage.voxa.userId;
+    } else {
+      userId = v1();
     }
 
+    _.set(this.dialogflow.conv.user.storage, "voxa.userId", userId);
```

Comment:
user storage is only available in google actions, let's move this to the related code to the relevant class